en/SUMMARY.md

```diff
@@ -2,28 +2,82 @@
+    * [Virtual Environment](chromebook/README.md#virtual-environment)
+    * [GitHub](chromebook/README.md#github)
+  * [Linux Installation](linux/README.md)
+    * [Command Line](linux/README.md#intro-command-line)
+    * [Python](linux/README.md#python)
+    * [Code Editor](linux/README.md#code-editor)
+    * [Virtual Environment](linux/README.md#virtualenv)
+    * [Django](linux/README.md#django)
+    * [Git](linux/README.md#git)
+    * [GitHub](linux/README.md#github-account)
+  * [Mac OSX Installation](macosx/README.md)
+    * [Command Line](macosx/README.md#intro-command-line)
+    * [Python](macosx/README.md#python)
+    * [Code Editor](macosx/README.md#code-editor)
+    * [Virtual Environment](macosx/README.md#virtualenv)
+    * [Django](macosx/README.md#django)
+    * [Git](macosx/README.md#git)
+    * [GitHub](macosx/README.md#github-account)
+    * [PythonAnywhere](macosx/README.md#pythonanywhere)
+  * [Windows Installation](windows/README.md)
+    * [Command Line](windows/README.md#intro-command-line)
+    * [Python](windows/README.md#python)
+    * [Code Editor](windows/README.md#code-editor)
+    * [Virtual Environment](windows/README.md#virtualenv)
+    * [Django](windows/README.md#django)
+    * [Git](windows/README.md#git)
+    * [GitHub](windows/README.md#github-account)
+    * [PythonAnywhere](windows/README.md#pythonanywhere-account)
 * [How the Internet works](how_the_internet_works/README.md)
 * [Introduction to command line](intro_to_command_line/README.md)
-* [Python installation](python_installation/README.md)
-* [Code editor](code_editor/README.md)
+  * [RunCode- Introduction to command line](runcode/intro_to_command_line/README.md)
+  * [ChromeBook/Linux- Introduction to command line](linux/intro_to_command_line/README.md)
+  * [Mac OSX - Introduction to command line](macosx/intro_to_command_line/README.md)
+  * [Windows - Introduction to command line](windows/intro_to_command_line/README.md)
+* [Python Installation](python_installation/README.md)
+  * [ChromeBook/Linux - Python installation](linux/python_installation/README.md)
+  * [Mac OSX - Python installation](macosx/python_installation/README.md)
+  * [Windows - Python installation](windows/python_installation/README.md)
 * [Introduction to Python](python_introduction/README.md)
+  * [RunCode - Introduction to Python](runcode/python_introduction/README.md)
+  * [Linux - Introduction to Python](linux/python_introduction/README.md)
+  * [Mac OSX - Introduction to Python](macosx/python_introduction/README.md)
+  * [Windows - Introduction to Python](windows/python_introduction/README.md)
+* [Code editor](code_editor/README.md)
 * [What is Django?](django/README.md)
-* [Django installation](django_installation/README.md)
+* [Django Installation](django_installation/README.md)
+  * [RunCode - Django installation](runcode/django_installation/README.md)
+  * [ChromeBook - Django installation](chromebook/README.md#glitch-cloud-ide)
```

Comment:
Here it points to the section, not README.MD, is it supposed to be?

Reply:
The Chromebook links have always been like that, pointing to a section in a README.md and not the README.md itself because they are in one file, though different applications are being installed. It's not a change I am introducing, and changing those links would introduce a new change. That's why I left them like that.